task 3.py

```diff
@@ -0,0 +1,7 @@
+x = list(map(int, list(iput())))
```

Comment:
Переводить введенную строку в список не обязательно, она ведь тоже является итерируемым объектом.
```suggestion
x = list(map(int, iput()))
```